src/renderer/components/TimelineSpace/Modals/NewToot/Status.vue

```diff
@@ -0,0 +1,178 @@
+        :key="index"
+        @click="insertAccount(item)"
+        @shortkey="insertAccount(item)"
+        @mouseover="highlightedIndex = index"
+        :class="{'highlighted': highlightedIndex === index}">
+        {{ item }}
+      </li>
+    </ul>
+  </el-popover>
+</div>
+</template>
+
+<script>
+import suggestText from '../../../../utils/suggestText'
+
+export default {
+  name: 'status',
+  props: {
+    value: {
+      type: String
+    },
+    opened: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      accounts: [
+        '@h3poteto@mstdn.io',
+        '@h3_poteto@friends.nico',
+        '@h3poteto@social.mikutter.hachune.net'
+      ],
+      filteredAccount: [],
+      openSuggest: false,
+      highlightedIndex: 0
+    }
+  },
+  computed: {
+    status: {
+      get: function () {
+        return this.value
+      },
+      set: function (value) {
+        this.$emit('input', value)
+      }
+    }
+  },
+  watch: {
+    opened: function (newState, oldState) {
+      if (!oldState && newState) {
+        this.$nextTick(function () {
+          this.$refs.status.focus()
+        })
+      }
+    }
+  },
+  methods: {
+    suggestAccount (e) {
+      // e.target.sectionStart: Cursor position
+      // e.target.value: current value of the textarea
+      const [start, word] = suggestText(e.target.value, e.target.selectionStart, '@')
+      if (!start || !word) {
+        return false
+      }
+      this.filteredAccount = this.accounts.filter((a) => a.startsWith(word))
+      if (this.filteredAccount.length === 0) {
+        return false
+      }
+      this.openSuggest = true
+    },
+    insertAccount (account) {
+      console.log(account)
+      this.openSuggest = false
+    },
+    suggestHighlight (index) {
+      if (index < 0) {
+        this.highlightedIndex = 0
+      } else if (index >= this.filteredAccount.length) {
+        this.highlightedIndex = this.filteredAccount.length - 1
+      } else {
+        this.highlightedIndex = index
+      }
+    },
+    selectAccount () {
+      console.log(this.highlightedIndex)
+      this.openSuggest = false
+    },
+    handleKey (event) {
+      switch (event.srcKey) {
+        case 'up':
+          this.suggestHighlight(this.highlightedIndex - 1)
+          break
+        case 'down':
+          this.suggestHighlight(this.highlightedIndex + 1)
+          break
+        case 'enter':
+          this.selectAccount()
+          return true
+        default:
+          return true
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.status {
+  textarea {
+    display: block;
+    padding: 5px 15px;
+    line-height: 1.5;
+    box-sizing: border-box;
+    width: 100%;
+    font-size: inherit;
+    color: #606266;
+    background-image: none;
+    border: 0;
+    border-radius: 4px;
+    resize: none;
+    height: 120px;
+    transition: border-color .2s cubic-bezier(.645,.045,.355,1);
```

Comment:
__[stylelint]__

Expected single space after "," in a single-line function (function-comma-space-after)


<!-- Comment by [Sider](https://sider.review) -->